src/Controller/ConfigController.php

```
@@ -170,4 +174,45 @@ public function update(Request $request, Session $session)
+        $suffix = $request->get('suffix');
+        $password = $request->get('password');
+        $port = $request->get('port');
+
+        $port = filter_var($port, FILTER_SANITIZE_NUMBER_INT);
+
+        // Connexion au serveur LDAP
+        $url = $protocol.'://'.$host.':'.$port;
+
+        $return = ["error"];
+
+        if ($fp=@fsockopen($host, $port, $errno, $errstr, 5)) {
```

PHPMD (2.15.0)

Avoid unused local variables such as '$fp'. (undefined)

(UnusedLocalVariable)